pages/api/book/event.ts

```diff
@@ -464,6 +466,13 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   log.debug(`Booking ${user.username} completed`);
 
+  const eventTrigger = rescheduleUid ? "BOOKING_RESCHEDULED" : "BOOKING_CREATED";
+  // Send Webhook call if hooked to BOOKING_CREATED & BOOKING_RESCHEDULED
+  const subscriberUrls = await getSubscriberUrls(user.id, eventTypeId, eventTrigger);
```

Comment:
What happens if some user puts an invalid URL here? Let's say someone puts `localhost` in there, the call will fail and the booking will fail also? How would we handle these kinds of events? @KATT @PeerRich @baileypumfleet

Reply:
@zomars has a point here, besides invalid URL - what if the user puts in something here that they expect to work but is invalid.

Reply:
@emrysal That is part of the next/follow-up PRs where we'd put a live test fire when the user creates a new webhook or edits the webhook subscriber URL.
@zomars Good point. The booking wouldn't fail since the webhook firing is placed "after" the booking is already completed. Thereafter, we're not doing anything with the response of the webhook call, so the system doesn't "fail" per say, but I do agree that we need to handle this possibility better.
It will be added as part of a series of follow-up PRs improving and completing this feature as a whole.